src/popups/HandicapPopup/HandicapPopup.scss

```diff
@@ -1,5 +1,5 @@
 .wkp-handicap-popup {
-  min-width: 350px;
+  min-width: 350px !important;
```

Comment:
An alternative would be to set the width instead of the min-width as in the private topic infofpw.